lib/ansible/modules/packaging/os/pulp_repo.py

```diff
@@ -561,9 +561,9 @@ def main():
     add_export_distributor = module.params['add_export_distributor']
     feed = module.params['feed']
     generate_sqlite = module.params['generate_sqlite']
-    importer_ssl_ca_cert = module.params['ca_cert']
-    importer_ssl_client_cert = module.params['client_cert']
-    importer_ssl_client_key = module.params['client_key']
+    importer_ssl_ca_cert = module.params['feed_ca_cert']
+    importer_ssl_client_cert = module.params['feed_client_cert']
```

Comment:
```suggestion
    importer_ssl_client_cert = module.params['feed_client_cert']
    if importer_ssl_client_cert is None and module.params['client_cert'] is not None:
      importer_ssl_client_cert = module.params['client_cert']
      module.deprecate("In Ansible 2.10 a new `peer_client_cert` option was added.  Until Ansible 2.14 the default value will come from the client_cert option", version="2.14")
```

Reply:
here's also a accidentally peer in the deprecate :)